docs/installing_cpython.md

```diff
@@ -158,6 +158,20 @@ and installing the free-threaded binaries is also possible:
 
     On macOS, the Python framework built with the free-threading ABI can be found at `$(brew --prefix)/Frameworks/PythonT.framework`.
 
+## Installing a Jupyter Kernel
+
+While jupyter [does not currently support free-threaded
+Python](https://github.com/jupyter/notebook/issues/7512), you can use Jupyter
```

Comment:
If you're linking to issues, maybe Jupyter Lab needs one too? https://github.com/jupyter/notebook/issues/7512 is closest.

Reply:
👍 I think the jupyterlab one is a better canonical issue. We'll probably close the one on `jupyter/notebook` as a duplicate.